.github/workflows/config/spelling_allowlist.txt

```diff
@@ -1,35 +1,36 @@
+AFQMC
 API
-APIs
 AST
 Aer
 AlmaLinux
+APIs
```

Comment:
This reordering of the items in this file will cause the CI to fail. It is expecting the items to be the order that the `sort` command would output.